login/visitwebpage.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package login
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+
+	"github.com/juju/cmd"
+	"github.com/juju/identity/idmclient"
+	"github.com/juju/usso"
+	"gopkg.in/macaroon-bakery.v1/httpbakery"
+)
+
+// VisitWebPage returns a function which will allow authentication with usso
+// via the cli.
```

Comment:
s/cli/CLI/

Although I don't think (as I mentioned above) this package should be dependent on cmd.Context.